sdk/servicebus/azure-servicebus/azure/servicebus/_common/constants.py

```diff
@@ -101,10 +101,33 @@
 PROPERTIES_DEAD_LETTER_REASON = b'DeadLetterReason'
 PROPERTIES_DEAD_LETTER_ERROR_DESCRIPTION = b'DeadLetterErrorDescription'
 
-
 DEAD_LETTER_QUEUE_SUFFIX = '/$DeadLetterQueue'
 TRANSFER_DEAD_LETTER_QUEUE_SUFFIX = '/$Transfer' + DEAD_LETTER_QUEUE_SUFFIX
 
+# Distributed Tracing Constants
+
+TRACE_COMPONENT_PROPERTY = 'component'
+TRACE_COMPONENT = 'servicebus'
+
+TRACE_NAMESPACE_PROPERTY = 'az.namespace'
+TRACE_NAMESPACE = 'Azure.ServiceBus'
+
+SPAN_NAME_RECEIVE = TRACE_NAMESPACE + '.receive'
+SPAN_NAME_RECEIVE_DEFERRED = TRACE_NAMESPACE + '.receive_deferred'
+SPAN_NAME_PEEK = TRACE_NAMESPACE + '.peek'
+SPAN_NAME_SEND = TRACE_NAMESPACE + '.send'
+SPAN_NAME_SCHEDULE = TRACE_NAMESPACE + '.schedule'
```

Comment:
just for myself learning, we decided not to have tracing for `cancel_schedule`?

Reply:
Eventually we likely should, it was just not min-bar. (same for settlement).  I believe .net has those due to T1 but the others are focusing on send/receive for GA.  I can create a backlog item to follow up.

Reply:
[Created](https://github.com/Azure/azure-sdk-for-python/issues/15271)